### ISSUE TITLE
[High Priority Feature]: Private build URLs for testing.

### ISSUE THREAD

**Issue by Bennick323** [WonderInteractive/WonderPlatform#11]
### Contact Details

Nick@immersedgames.com

### Desired feature.

For our use case, we want to be able to embed our game from the wonder platform on our own domain. However, it seems like the only option to get the URL for the game is to set it as an "Active Revision" for the project, which I'm pretty sure will result in the game going live to the front page of the platform. We will need to be able to get a URL for a build of our game to test this embedding functionality without exposing a potentially unstable build of the game to the front page of the platform. 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by MrPurple6411** [comment]
Very good idea and use case that I have on my list of features I am actively working on as it is very needed to be able to do test builds. 

I will do my best to get this up and running as quickly as I can.

**Comment by MrPurple6411** [comment]
@Bennick323 

I have this system ready for testing on the development site.
The person trying to load the test page must be logged into our site so that the auth system can verify they are authorized to be loading these test revisions.

The format to get to said test launch page is `https://dev.theimmersiveweb.com/create/{Project Name}/{Revision Number}`
The Launch buttons on the dev projects pages now send you to this location as well so you can just use that and then copy the url from the address bar.

Please let me know how this works out for you and if you feel it requires any changes.

**Comment by Bennick323** [comment]
> 

So it seems to work for direct access, which is good. However it doesn't seem to like us attempting to embed the game using iframe in a webpage, which is what we're primarily trying to accomplish. The auth doesn't seem like it's carrying over.

**Comment by Bennick323** [comment]
@MrPurple6411 To be more specific, when we embed the test launch page into our test web app, it gives us this screen and this log message. 
![image](https://user-images.githubusercontent.com/16246462/163487068-114472ca-1f14-40c5-87f8-73a7c23467ec.png)
![image](https://user-images.githubusercontent.com/16246462/163487132-947b4d1e-eeea-4f45-a629-de98bea34b86.png)

We've also since tried enabling the "Dev Embed" checkbox that was added recently and got the same result. Just in case it was important, we've tried this with both Shipping and Development builds.

**Comment by MrPurple6411** [comment]
> @MrPurple6411 To be more specific, when we embed the test launch page into our test web app, it gives us this screen and this log message. ![image](https://user-images.githubusercontent.com/16246462/163487068-114472ca-1f14-40c5-87f8-73a7c23467ec.png) ![image](https://user-images.githubusercontent.com/16246462/163487132-947b4d1e-eeea-4f45-a629-de98bea34b86.png)
> 
> We've also since tried enabling the "Dev Embed" checkbox that was added recently and got the same result. Just in case it was important, we've tried this with both Shipping and Development builds.

Having an issue with a worker not updating. I should have this fixed today. 
I will let you know here when it is up and running, sorry for the delay.

**Comment by MrPurple6411** [comment]
@Bennick323 All right so we got the worker issue resolved and my testing is showing that it should now be functional

**Comment by Bennick323** [comment]
@MrPurple6411 Yeah, we see it working here! We are looking into some other tangentially-related issues on our side. I will let you know if we run into anything else. Thank you!
